satellite/repair/repairer/ec.go

```diff
@@ -70,6 +70,8 @@ func (ec *ECRepairer) Get(ctx context.Context, limits []*pb.AddressedOrderLimit,
 		return nil, nil, Error.New("number of non-nil limits (%d) is less than required count (%d) of erasure scheme", nonNilCount(limits), es.RequiredCount())
 	}
 
+	mon.IntVal("ECRepairer_Get_nonNilLimits").Observe(int64(nonNilLimits))
```

Comment:
With most other metrics, we've had an all lowercase convention. I think we should stick with that.
Also, would it make more sense to send this metric or its equivalent from repairer/segments.go? Then you could just send `len(healthyPieces)`, which should be the same number as "non nil limits" for Get